### PR TITLE
HDDS-1425. Ozone compose files are not compatible with the latest docker-compose

### PR DESCRIPTION
**Pull request by elek**
I upgraded my docker-compose to the latest available one (1.24.0)

But after the upgrade I can't start the docker-compose based cluster any more:

```
./test.sh 
-------------------------------------------------
Executing test(s): [basic]

  Cluster type:      ozone
  Compose file:      /home/elek/projects/hadoop-review/hadoop-ozone/dist/target/ozone-0.4.0-SNAPSHOT/smoketest/../compose/ozone/docker-compose.yaml
  Output dir:        /home/elek/projects/hadoop-review/hadoop-ozone/dist/target/ozone-0.4.0-SNAPSHOT/smoketest/result
  Command to rerun:  ./test.sh --keep --env ozone basic
-------------------------------------------------
ERROR: In file /home/elek/projects/hadoop-review/hadoop-ozone/dist/target/ozone-0.4.0-SNAPSHOT/compose/ozone/docker-config: environment variable name 'LOG4J2.PROPERTIES_appender.rolling.file 
```

It turned out that the line of LOG4J2.PROPERTIES_appender.rolling.file contains an unnecessary space which is not accepted by the latest docker-compose any more.

See: https://issues.apache.org/jira/browse/HDDS-1425

### CODE REVIEW

File: hadoop-ozone/dist/src/main/compose/ozone-recon/docker-config [hunk 1]
```diff
@@ -60,7 +60,8 @@ LOG4J2.PROPERTIES_appender.console.layout.type=PatternLayout
 LOG4J2.PROPERTIES_appender.console.layout.pattern=%d{DEFAULT} | %-5level | %c{1} | %msg | %throwable{3} %n
 LOG4J2.PROPERTIES_appender.rolling.type=RollingFile
 LOG4J2.PROPERTIES_appender.rolling.name=RollingFile
-LOG4J2.PROPERTIES_appender.rolling.fileName =${sys:hadoop.log.dir}/om-audit-${hostName}.log
+LOG4J2.PROPERTIES_appender.rolling.fileName
+=${sys:hadoop.log.dir}/om-audit-${hostName}.log
```
Comment (adoroszlai) [inline hadoop-ozone/dist/src/main/compose/ozone-recon/docker-config]: Looks like newline is unintended.

Comment (elek) [reply hadoop-ozone/dist/src/main/compose/ozone-recon/docker-config]: oh, thanks. You saved me. Fixed.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 28 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1128 | trunk passed |
| +1 | compile | 69 | trunk passed |
| +1 | mvnsite | 26 | trunk passed |
| +1 | shadedclient | 639 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 23 | trunk passed |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 25 | dist in the patch failed. |
| +1 | compile | 33 | the patch passed |
| +1 | javac | 33 | the patch passed |
| +1 | mvnsite | 35 | the patch passed |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | shelldocs | 16 | The patch generated 0 new + 104 unchanged - 132 fixed = 104 total (was 236) |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 707 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 15 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 19 | dist in the patch passed. |
| +1 | asflicense | 27 | The patch does not generate ASF License warnings. |
| | | 2920 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-727/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/727 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  shellcheck  shelldocs  |
| uname | Linux f25d9821457b 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / abace70 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| shellcheck | v0.4.6 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-727/1/artifact/out/patch-mvninstall-hadoop-ozone_dist.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-727/1/testReport/ |
| Max. process+thread count | 457 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist U: hadoop-ozone/dist |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-727/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by bharatviswa504** [comment]
+1 LGTM.
Thank You @elek for the fix.
I will commit this.

**Pull request merged**